taskvine/src/manager/vine_manager.c

```diff
@@ -554,11 +554,13 @@ static vine_result_code_t get_completion_result(struct vine_manager *q, struct v
 			original->library_failed_count++;
 			original->time_when_last_failure = timestamp_get();
 		}
+		itable_remove(q->running_library_instances, t->task_id);
```

Comment:
It makes sense to have a running library table.

But let's put this down at line 617 to keep all the core data structure changes together.

(`itable_remove` will be a no-op if it is not a library task.)